packages/react/src/index.ts

```diff
@@ -10,6 +10,7 @@ export * from "./input";
 export * from "./kbd";
 export * from "./paper";
 export * from "./progress";
+export * from "./radioGroup";
```

Comment:
let's use `kebab-case` for directories

```suggestion
export * from "./radio-group";
```